### PR TITLE
Setup CI

### PR DESCRIPTION
**Pull request by d-staehler**

### CODE REVIEW

**Comment by steffanlevet** [comment]
Task linked: [CU-2av96b Set up CI and hex publishing for mix_deploy](https://app.clickup.com/t/2av96b)

File: test/mix_deploy_test.exs [hunk 1]
```diff
@@ -1,8 +1,2 @@
 defmodule MixDeployTest do
```
Comment (jimsynz) [inline test/mix_deploy_test.exs]: May as well delete this whole file.

**Review by jimsynz — APPROVED** [review]
banderas says "awesome"

File: test/mix_deploy_test.exs (hunk 1 above)
Comment (d-staehler) [reply test/mix_deploy_test.exs]: Good point 👍

**Pull request merged**